src/disc/Albany_ExtrudedDiscretization.cpp

```diff
@@ -86,61 +86,10 @@ ExtrudedDiscretization::setupMLCoords()
       getOverlapVectorSpace());
 }
 
-void
-ExtrudedDiscretization::writeSolution(
-    const Thyra_Vector& /* soln */,
-    const Teuchos::RCP<const Thyra_MultiVector>& /* soln_dxdp */,
-    const double        /* time */,
-    const bool          /* overlapped */,
-    const bool          /* force_write_solution */)
-{
-  printf("WARNING ExtrudedDiscretization::writeSolution not yet implemented\n");
-  // throw NotYetImplemented("ExtrudedDiscretization::writeSolution");
-}
-
-void
-ExtrudedDiscretization::writeSolution(
-    const Thyra_Vector& /* soln */,
-    const Teuchos::RCP<const Thyra_MultiVector>& /* soln_dxdp */,
-    const Thyra_Vector& /* soln_dot */,
-    const double        /* time */,
-    const bool          /* overlapped */,
-    const bool          /* force_write_solution */)
-{
-  printf("WARNING ExtrudedDiscretization::writeSolution not yet implemented\n");
-  // throw NotYetImplemented("ExtrudedDiscretization::writeSolution");
-}
-
-void
-ExtrudedDiscretization::writeSolution(
-    const Thyra_Vector& /* soln */,
-    const Teuchos::RCP<const Thyra_MultiVector>& /* soln_dxdp */,
-    const Thyra_Vector& /* soln_dot */,
-    const Thyra_Vector& /* soln_dotdot */,
-    const double        /* time */,
-    const bool          /* overlapped */,
-    const bool          /* force_write_solution */)
-{
-  printf("WARNING ExtrudedDiscretization::writeSolution not yet implemented\n");
-  // throw NotYetImplemented("ExtrudedDiscretization::writeSolution");
-}
-
-void
-ExtrudedDiscretization::writeSolutionMV(
-    const Thyra_MultiVector& /* soln */,
-    const Teuchos::RCP<const Thyra_MultiVector>& /* soln_dxdp */,
-    const double             /* time */,
-    const bool               /* overlapped */,
-    const bool               /* force_write_solution */)
-{
-  throw NotYetImplemented("ExtrudedDiscretization::writeSolutionMV");
-}
-
 void
 ExtrudedDiscretization::writeSolutionToMeshDatabase(
     const Thyra_Vector& /* soln */,
     const Teuchos::RCP<const Thyra_MultiVector>& /* soln_dxdp */,
-    const double /* time */,
```

Comment:
Why are you removing time info here?

Reply:
Because the time var is only used when writing _to file_, to assign a label to the unlimited time dim (in netcdf lingo). When we write to the in-memory mesh database, there is no concept of time: the mesh only stores ONE copy of the solution field at any given moment...

Reply:
To clarify a bit better, saving the solution has 2 phases: saving to the in-memory mesh data structures, and writing to file. The time value is only needed in the latter.